app/helpers/application_helper/button/vm_reset.rb

```diff
@@ -0,0 +1,13 @@
+class ApplicationHelper::Button::VmReset < ApplicationHelper::Button::Basic
+  needs_record
+
+  def skip?
+    !@record.is_available?(:reset)
+  end
+
+  def disable?
+    if @record.is_available_now_error_message(:reset)
+      return @record.is_available_now_error_message(:reset)
```

Comment:
hmm, thats really strange, because a `?` method suffix should always return a boolean. @romanblanco do you mind investigating further if this could be changed to return only a boolean?